soccer/gameplay/plays/offense/kick_gently.py

```diff
@@ -0,0 +1,96 @@
+import play
+import behavior
+import skills.move
+import skills.pivot_kick
+import constants
+import robocup
+import main
+import enum
+import timeout_behavior
+import tactics.coordinated_pass
+import evaluation.touchpass_positioning
+
+
+class KickGently(play.Play):
```

Comment:
The issue appears to be something with either pivot kicking a ball moving away from the robot or with kicking very gently. Pivot kick goes into kicking but the ball doesn't actually move away from the robot at all. I know 10% power is enough to move the ball because the first kick works but then the rest of the kicks don't.